examples/deployment/haproxy/README.md

```diff
@@ -0,0 +1,107 @@
+# Deploying HAProxy Ingress Controller
+
+Don't have a Kubernetes cluster? Single-node of [CoreOS Kubernetes](https://github.com/coreos/coreos-kubernetes/) is a good starting point.
+
+Deploy a default backend used to serve `404 Not Found` pages:
+
+    kubectl run ingress-default-backend \
```

Comment:
we usually use `console` quotes, eg: 
https://raw.githubusercontent.com/jcmoraisjr/ingress/b579cbba7412bc6999bbfd168c60c80c4da1be75/examples/deployment/nginx/README.md